jumpscale/servers/threebot/threebot.py

```diff
@@ -142,6 +142,7 @@ def apply(self):
                         loc.port = location.get("port", PORTS.HTTP)
                         loc.path_dest = location.get("path_dest", "")
                         loc.websocket = location.get("websocket", False)
+                        loc.increase_buffers_size = location.get("increase_buffers_size", False)
```

Comment:
If you add a property, it should be added to `Location class too, this way, it won't be stored.